src/SmartComponents/SystemsTable/SystemsTable.js

```diff
@@ -115,7 +114,7 @@ export const SystemsTable = ({
                     } }
                 bulkSelect={ onSelect && !isAddSystemNotifications && {
                     isDisabled: !hasMultiSelect,
-                    count: entities && entities.selectedSystemIds ? entities.selectedSystemIds.length : 0,
+                    count: entities?.selectedSystems?.length,
```

Comment:
This shows only selected items on the viewed page and not the whole selection.

![Screenshot_2021-06-08 Comparison - Drift Red Hat Insights](https://user-images.githubusercontent.com/7695766/121150688-07418d80-c844-11eb-9548-9f15470bfec3.png)
(here the bulk select is completely empty, check the selection on top right)

Reply:
After I got an error (selecting 51 systems) for comparison and opening the modal again, I see the selection is visible in the basked however it is not displayed on the table nor on the bulk select.

![Screenshot_2021-06-08 Comparison - Drift Red Hat Insights(1)](https://user-images.githubusercontent.com/7695766/121151492-b3837400-c844-11eb-93ed-d090ce10c7fc.png)
(I had first page selected and then one item on next page)

Reply:
> This shows only selected items on the viewed page and not the whole selection.
> 
> ![Screenshot_2021-06-08 Comparison - Drift Red Hat Insights](https://user-images.githubusercontent.com/7695766/121150688-07418d80-c844-11eb-9548-9f15470bfec3.png)
> (here the bulk select is completely empty, check the selection on top right)

If I understand it correctly, this is what we expect. If you make a full selection on page 1, and then go to page 2, if nothing on page 2 is selected you should see the number of selected from page 1 in the selection in the top right, but the bulk select should be unchecked.

Reply:
We implemented the bulk select recently, and the behavior we used was the one I've described (whole selection). What is the source of the expectation for the bulk to display current page selection? It feels a bit strange TBH.
Maybe @bastilian can chime in?

Reply:
none selected = empty check
some selected = dash
all selected = check

I did review with UX, and they were okay with the use case.

Reply:
Not sure what I should chime in about or how. :)

The bulk select with the selection of all items indicated outside and only having page selection operations, is different from how we implemented it on our rules table, but if UX approves of it it should be ok.

Reply:
@johnsonm325 If UX agrees, then it is all good. 
Can you please check the issue described in the second comment https://github.com/RedHatInsights/drift-frontend/pull/481#discussion_r647232493?